.github/workflows/ci.yml

```diff
@@ -171,14 +171,10 @@ jobs:
         check: ${{ fromJson(needs.build-info.outputs.basic-checks) }}
       fail-fast: false
     steps:
-      - name: Checkout project
-        uses: actions/checkout@v3
-        if: matrix.check != 'bats'
       - name: Checkout project with history
         uses: actions/checkout@v3
         with:
           fetch-depth: 0
-        if: matrix.check == 'bats'
```

Comment:
If I understand correctly, only `unit` check needs git history (in addition to `bats`).  So instead of removing the if conditions, please add `unit` to them.

Reply:
done